### ISSUE TITLE
12 PM is incorrectly outputting as AM

### ISSUE THREAD

**Issue by compuvin** [compuvin/Format-VBscript-ASP#1]
12 noon is incorrectly outputting as AM. The line of code that is causing this is this if statement:

`if instr(1,sFormat,"AM/PM") > 0 and int(hour(vExpression)) > 12 then
	    nExpression = replace(nExpression,"HH",right("00" & hour(vExpression)-12,2)) '2 character hour
	    nExpression = replace(nExpression,"H",hour(vExpression)-12) '1 character hour
		nExpression = replace(nExpression,"AM/PM","PM") 'If if its afternoon, its PM
	  else
	    nExpression = replace(nExpression,"HH",right("00" & hour(vExpression),2)) '2 character hour
	    nExpression = replace(nExpression,"H",hour(vExpression)) '1 character hour
		nExpression = replace(nExpression,"AM/PM","AM") 'If its not PM, its AM
	  end if`

Basically if the hour is 12, it moves to the "else" line of the code and proceeds as AM.
